bower.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "purescript-leaflet",
+  "name": "purescript-leafletjs",
```

Comment:
Oh, I see from the other PR it was. We should perhaps rename this repo to match the bower name then?